blog_system/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from django.urls import path, include
+from debug_toolbar.toolbar import debug_toolbar_urls
```

Comment:
It seems like you're trying to import `debug_toolbar_urls` from `debug_toolbar.toolbar`, but there's no such module or function in Django Debug Toolbar. If you're trying to include Django Debug Toolbar in your URLs, you should do it like this: `path('__debug__/', include(debug_toolbar.urls))` in your urlpatterns, and make sure to import `debug_toolbar` at the top of your file.